README.md

```diff
@@ -21,13 +21,31 @@ If you wish to see visualisations of parse trees, you may optionally install:
 * PyDot https://code.google.com/p/pydot/
 * Pygame https://www.pygame.org/
 
-
 ### Running Eco ###
 
 To run Eco, use the bin/eco file:
 
   `$ bin/eco`
-  
+
 ### Tutorial ###
 
 A small tutorial to get you started with the basics of Eco can be found [here](tutorial/TUTORIAL.md).
+
+### Troubleshooting ###
+
+#### Windows Subsystem for Linux running Ubuntu ####

```

Comment:
@ptersilie I am pretty sure this also works on and is applicable for native Ubuntu.